.github/workflows/canary-integration-test.yml

```diff
@@ -390,11 +390,14 @@ jobs:
         if: always()
         uses: ./.github/workflows/collect-logs
         with:
-          name: canary
+          name: ${{ github.job }}-${{ matrix.ceph-image }}
```

Comment:
There were several instances of jobs collecting logs named `canary`. This would mean that only one of the jobs' outputs would actually be collected, which is no bueno. I think using `${{ github.job }}` is a really good antidote to this problem for the future. 

And here I'm also adding the matrix value because that is important for matrix'ed runs.